examples/get_projects.py

```diff
@@ -42,7 +44,35 @@ def sample_get_projects():
     else:
         return p
 
+def sample_get_project_by_id():
+    url = os.environ.get('FLN_URL')
+    oauth_token = os.environ.get('FLN_OAUTH_TOKEN')
+    session = Session(oauth_token=oauth_token, url=url)
+
+    project_id = 2
+    project_details = create_get_projects_project_details_object(
+        full_description=True,
+        tracks=True
```

Comment:
I don't think we should add `tracks` projection in a sample.

If you want, you can add a new sample for showing how we can get tracks.